apps/sms/views/shared/js/navigation.js

```diff
@@ -583,24 +583,35 @@ function switchPanel() {
   var isGoingBack = oldView && oldView.previous === newView.name;
 
   newPanelElement.style = '';
+  newPanelElement.classList.remove('panel-hidden');
+  newPanelElement.classList.add('panel-active');
 
   var doSlideAnimation =
     from && from.panel &&
     from.panel.behavior !== newView.behavior;
 
+  var doFadeInAnimation = !from || !from.panel;
+
   var animationPromise;
   if (doSlideAnimation) {
-
     var shouldGoRight = isGoingBack ^ (document.dir === 'rtl');
     newPanelElement.style.animationName =
-      shouldGoRight ? 'new-slide-right' : 'new-slide-left';
+      shouldGoRight ? 'left-to-current' : 'right-to-current';
     oldPanelElement.style.animationName =
-      shouldGoRight ? 'old-slide-right' : 'old-slide-left';
+      shouldGoRight ? 'current-to-right' : 'current-to-left';
```

Comment:
I'm fine with using the same approach with animation-name.

__issue:__ Seems there is a flaw somewhere here - try to send group message and you'll end up in the broken Report view. Just checked and see that intergration tests caught something like this.